src/main/java/com/google/cloud/tools/appengine/api/devserver/RunConfiguration.java

```diff
@@ -53,6 +54,13 @@
 
   List<String> getJvmFlags();
 
+  /**
+   * Return an immutable map of environment variable strings.
+   * 
+   * @return the environment variables
+   */
+  ImmutableMap<String, String> getEnvironmentVariables();
```

Comment:
What's strange is that ImmutableMap is a subclass of Map rather than the other way around. It's a violation of the Liskov substitution principal, but that's Java for you. :-(